polkit: No system-auth in OE-Core, we can use common-* in place of it.

Upstream-Status:Inappropriate [configuration]

Signed-off-by: Xiaofeng Yan <xiaofeng.yan@windriver.com>

Upstream-Status: Inappropriate [oe specific]
Rebase to 0.115
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 36df239..8b3e1b1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -471,10 +471,10 @@ elif test x$with_os_type = xfreebsd -o x$with_os_type = xnetbsd; then
     PAM_FILE_INCLUDE_PASSWORD=system
     PAM_FILE_INCLUDE_SESSION=system
 else
-   PAM_FILE_INCLUDE_AUTH=system-auth
-   PAM_FILE_INCLUDE_ACCOUNT=system-auth
-   PAM_FILE_INCLUDE_PASSWORD=system-auth
-   PAM_FILE_INCLUDE_SESSION=system-auth
+   PAM_FILE_INCLUDE_AUTH=common-auth
+   PAM_FILE_INCLUDE_ACCOUNT=common-account
+   PAM_FILE_INCLUDE_PASSWORD=common-password
+   PAM_FILE_INCLUDE_SESSION=common-session
 fi
 
 AC_SUBST(PAM_FILE_INCLUDE_AUTH)
-- 
2.7.4

